instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/SqsImpl.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v1_11;
+
+import com.amazonaws.AmazonWebServiceRequest;
+import com.amazonaws.Request;
+import com.amazonaws.Response;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.Message;
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest;
+import com.amazonaws.services.sqs.model.ReceiveMessageResult;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+
+final class SqsImpl {
+  static {
+    // Force loading of SQS class; this ensures that an exception is thrown at this point when the
+    // SQS library is not present, which will cause SqsAccess to have enabled=false in library mode.
+    @SuppressWarnings("unused")
+    String ensureLoadedDummy = AmazonSQS.class.getName();
+  }
+
+  private SqsImpl() {}
+
+  public static boolean afterResponse(
+      Request<?> request,
+      Response<?> response,
+      Instrumenter<Request<?>, Response<?>> consumerInstrumenter) {
+    if (response.getAwsResponse() instanceof ReceiveMessageResult) {
```

Comment:
Previously we instead checked that the request is a ReceiveMessageRequest. This should not matter, but since we cast the response w/o expecting exceptions, I thought it safer to check the response instead.